addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17814,7 +17820,29 @@ msgctxt "#36614"
 msgid "Show add-ons that are currently running in the background."
 msgstr ""
 
-#empty strings from id 36615 to 36899
+#: system/settings/settings.xml
```

Comment:
mind adding a contextual note before merging? Something like:
`#. Label of a setting or option referring to the source/origin of f.e. Add-ons`
Just to give translators a hint in which context "sources" is used (we also have media sources to populate the library). Thanks.